src/ButtonBack/ButtonBack.jsx

```diff
@@ -83,7 +86,7 @@ export default class ButtonBack extends React.Component {
     );
 
     return (
-      <button
+      <Tag
         type="button"
```

Comment:
This isn't valid on all HTML elements. For example, you can't use the `type="button"` on `<div>` tags. Also, you will run into accessibility problems if it is an interactive element and it uses non-interactive HTML (i.e., `<p>` or `<div>`).

I would be curious as to people's thoughts on the above, especially considering accessibility. On one hand, we are acknowledging that there could be an accessibility problem. On the other hand, preventing it forces an opinion - and maybe someone wants to break accessibility. I am not sure, but I think some conversation would be warranted.

Reply:
I agree, it will rely on the developer who's implementing the package to do it properly.

I would personally recommend to explain this limitation/possible issues in the documentation, and give the recommendation to use a button as the outputed HTML element, because of the said arguments (a11y, W3C compliance, etc.), maybe with a generic example using a FancyButton component

That will make it clear that the native usage is still a11y/w3c compliant, and the generic options may break that compliance if wrongly implemented.

Reply:
I am open to that - could we add that warning to the documentation on the `tag` property. Something along the lines of WARNING: Failure to use a button or an interactive DOM element with the appropriate props can result in failed a11y/w3c compliance.

Reply:
I added the warning on all tag property descriptions